backend/btrixcloud/models.py

```diff
@@ -1327,6 +1482,9 @@ class CollOut(BaseMongoModel):
 
     allowPublicDownload: bool = True
 
+    pages: List[PageOut] = []
+    pagesQuery: str = ""
```

Comment:
I might suggest remaining this to `pagesQueryUrl` to be a bit clearer on what it is, otherwise could read like it's some representation of the query that produced this result.

Reply:
renamed to pagesQueryUrl